terraform/fullnodes.tf

```diff
@@ -19,7 +19,7 @@ resource "aws_instance" "fullnode" {
   user_data                   = local.user_data
 
   dynamic "root_block_device" {
-    for_each = contains(local.ebs_types, split(var.validator_type, ".")[0]) ? [0] : []
+    for_each = contains(local.ebs_types, split(".", var.validator_type)[0]) ? [0] : []
```

Comment:
would typically recommend splitting this into two commits but seems harmless enough.